frontend/ui/src/components/AiryWebsocket/index.tsx

```diff
@@ -28,7 +29,8 @@ const mapDispatchToProps = dispatch => ({
   addMessages: (conversationId: string, messages: Message[]) => dispatch(addMessagesAction({conversationId, messages})),
   onChannel: (channel: Channel) => dispatch(setChannelAction(channel)),
   getConversationInfo: (conversationId: string) => dispatch(getConversationInfo(conversationId)),
-  onMetadata: (metadataEvent: MetadataEvent) => dispatch(setMetadataAction(metadataEvent)),
+  onMetadata: (metadataEvent: MetadataEvent) =>
+    dispatch(setMetadataAction(camelcaseKeys(metadataEvent, {deep: true, stopPaths: ['metadata.userData']}))),
```

Comment:
I would have expected `metadata.user_data` here... but maybe I don't get the api?